frontend/src/modules/entitydetails/components/EntityNavigation.js

```diff
@@ -28,6 +29,10 @@ export default class EntityNavigation extends React.Component<Props> {
         document.removeEventListener('keydown', this.handleShortcuts);
     }
 
+    copyLinkToClipboard = () => {
+        this.props.copyLinkToClipboard();
+    }
+
```

Comment:
I would actually not create this function and directly use `this.props.copyLinkToClipboard` below. For the same reason, I'd also remove the `goToNextEntity()` and `goToPreviousEntity()` and make sure we always use `this.props.goToNextEntity` and `this.props.goToPreviousEntity`.